go/ops.go

```diff
@@ -166,6 +166,10 @@ func (op *InnerOp) CheckAgainstSpec(spec *ProofSpec, b int) error {
 		return errors.New("spec.InnerSpec.ChildSize must be >= 1")
 	}
 
+	if spec.InnerSpec.MaxPrefixLength >= spec.InnerSpec.MinPrefixLength+spec.InnerSpec.ChildSize {
```

Comment:
for what it is worth. Spec fields being validated against themselves are better off in a spec.Validate() function, but for the sake of minimizing changes, this approach makes sense to me